### DISCUSSION TITLE
Add HTTP verbs support for invokable classes?

### DISCUSSION THREAD

**Discussion by sikhlana** [spatie/laravel-route-attributes#52] [Ideas]

**Comment by osbre** [comment]
I'm using Invokable Controller and it works as expected.
Example: https://github.com/smuzi-ua/dumka-backend/blob/c8ad89a66e3cfcce7772649a203ebd219dbf88d3/app/Http/Controllers/API/StudentVerificationController.php

**Comment by sikhlana** [comment]
@osbre that would obviously work but not what I am talking about.

If a controller is invokable, adding something like this should also work:

```php
#[Get('something')]
class SomeController extends Controller
{
    public function __invoke()
    {
        return response()->json([]);
    }
}
```

This just looks cleaner and easily attainable.
